FAQUse: What is difference anonset and anonscore (#1392)
zkSNACKs/WasabiDoc: docs/FAQ/FAQ-UseWasabi.md

For example, if the outputs are [1, 1, 1, 1, 0.5, 0.5, 0.5, 0.5, 0.4, 0.4, 0.2, 0.2, 0.2, 0.2, 0.2, 0.1, 0.1, 0.1, 0.1, 0.1] then, even when each of those 1s have anonscore 4, there are still lots of combinations of outputs that sum up to 1, so the anonset should be much higher but the anonscore is extremely conservative and takes into account many other things.
:::

:::details
### What is the difference between anonymity set and anonymity score?

The anonscore evolved into existence from the anonset, but with some differences.
What differs is the kind of coinjoins they are applied to, where anonset was for Wasabi 1 and anonscore is for current Wasabi 2.
In Wasabi 1 coinjoins there are unequal inputs, some equal outputs, and change coins. 

> In Wasabi 2 coinjoins there are many equal inputs, many equal outputs, and (sometimes) some unique value outputs, but that does not mean that these unique value outputs are change coins. 

Reviewer: ```suggestion
In Wasabi 2 coinjoins there are many equal inputs, many equal outputs, and (sometimes) some unique value outputs, that are not necessarily change coins.
```

> In fact most of the times they are well mixed coins.

Reviewer: ```suggestion
In fact most of the times they are well mixed coins.

Basically the anonymity score is an extremely conservative anonymity set calculation.
```

For example, if the outputs are [1, 1, 1, 1, 0.5, 0.5, 0.5, 0.5, 0.4, 0.4, 0.2, 0.2, 0.2, 0.2, 0.2, 0.1, 0.1, 0.1, 0.1, 0.1] then, even when each of those 1s have anonscore 4, there are still lots of combinations of outputs that sum up to 1, so the anonset should be much higher but the anonscore is extremely conservative and takes into account many other things.
:::

:::details

> ### What is the difference between anonset and anonscore?

Reviewer: ```suggestion
### What is the difference between anonymity set and anonymity score?
```

> The anonset and anonscore have minimal differences.
> The anonscore evolved into existence from the anonset, but with minimal changes, so they can be considered roughly the same thing.

Reviewer: ```suggestion
The anonscore evolved into existence from the anonset, but with some differences.
```

What differs is the kind of coinjoins they are applied to, where anonset was for Wasabi 1 and anonscore is for current Wasabi 2.
In Wasabi 1 coinjoins there are unequal inputs, some equal outputs, and change coins. 

> In Wasabi 2 coinjoins there are many equal inputs, equal outputs, and (sometimes) some unique value outputs, but that does not mean that these unique value outputs are a change coin. 

Reviewer: ```suggestion
In Wasabi 2 coinjoins there are many equal inputs, many equal outputs, and (sometimes) some unique value outputs, but that does not mean that these unique value outputs are change coins. 
```

> In fact most of the times it's not a change, but a private well mixed coin.

Reviewer: ```suggestion
In fact most of the times they are well mixed coins.
```
